### DISCUSSION TITLE
How to add nodejs?

### DISCUSSION THREAD

**Discussion by zt1106** [Darin755/browser-linux#12] [Q&A]
I added   BR2_PACKAGE_NODEJS=y but It failed to build..

**Comment by Darin755** [comment]
That should've worked. Can you give me the console output?

**Comment by Darin755** [comment]
To install in the base image run:
```
sudo udhcpc
static-get --install --verbose opt-nodejs-0.8.18-1
```

keep in mind sudo is a little temperamental when its run for the first time (I don't know why)
